docker/server.py

```diff
@@ -0,0 +1,32 @@
+from aiohttp import web
+import os
+import uuid
+from cellpack.bin.pack import pack
+
+SERVER_PORT = 8443
+
+async def hello_world(request: web.Request) -> web.Response:
+    return web.Response(text="Hello world")
+
+async def pack_handler(request: web.Request) -> web.Response:
+    recipe = request.rel_url.query.get("recipe")
+    if recipe is None:
+        # pack request must include "recipe" query parameter
+        raise web.HTTPBadRequest()
+    config = request.rel_url.query.get("config")
+    job_id = str(uuid.uuid4())
+    os.environ["AWS_BATCH_JOB_ID"] = job_id
+    pack(recipe=recipe, config_path=config, docker=True)
```

Comment:
should we add an exception here to prevent server crash if the pack function fails?